### PR TITLE
Issue/2111 Use ex unit as universal line-height definition

### PR DESCRIPTION
**Pull request by stephanschroeter**
… | #2111

### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://github.com/porscheofficial/oss-docs/blob/67c5b24838a293ce7a964884e6005eb71f2b8579/CONTRIBUTOR_LICENSE_AGREEMENT.md)

#### References

- Preview: 
  - https://designsystem.porsche.com/issue/2111/latin
  - https://designsystem.porsche.com/issue/2111/greek
  - https://designsystem.porsche.com/issue/2111/cyril
  - https://designsystem.porsche.com/issue/2111/chinese
  - https://designsystem.porsche.com/issue/2111/vietnamese
  - https://designsystem.porsche.com/issue/2111/korean
  - https://designsystem.porsche.com/issue/2111/japanese

#### Scope

Short description of the pull request scope

### CODE REVIEW

File: packages/components/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 ### [Unreleased]
 
+#### Changed
+
+- `line-height` calculation for all components is handled CSS only now by using `ex`-unit in combination with `calc()`
+  which gives the best performance, the easiest possible integration and respects UI best practices in having **larger**
+  `line-height` values for **small** `font-size` definitions and **smaller** `line-height` values for **larger**
+  `font-size` definitions. The calculated values by CSS slightly differ compared to the ones calculated by JavaScript,
+  which might result in minor visual changes, e.g.:
+  - 16/24px (JS) => 16/24px (CSS)
```
Comment (stephanschroeter) [inline packages/components/CHANGELOG.md:19]: add diff

File: packages/components/src/styles/link-button-pure-styles.ts [hunk 2]
```diff
@@ -162,7 +144,7 @@ export const getLinkButtonPureStyles = (
       border: 'none',
       background: 'transparent',
       color: isDisabledOrLoading ? disabledColor : active ? activeColor : baseColor,
-      transition: `${getTransition('color')}, font-size 1ms linear`, // used for transitionend event listener
+      transition: `${getTransition('color')}`,
```
Comment (stephanschroeter) [inline packages/components/src/styles/link-button-pure-styles.ts:147]: remove backticks

File: packages/components/src/utils/validation/partials/validatePartialUsage.ts [hunk 3]
```diff
@@ -20,9 +20,10 @@ export const validatePartialUsage = (): void => {
   if (ROLLUP_REPLACE_IS_STAGING !== 'staging' && process.env.NODE_ENV !== 'development') {
     validateGetInitialStylesUsage();
     validateFontFaceStylesheetUsage();
```
Comment (stephanschroeter) [inline packages/components/src/utils/validation/partials/validatePartialUsage.ts:22]: add comment `validateGetFontFaceStylesheetUsage`, naming wrong

File: packages/utilities-deprecated/package.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
   "license": "UNLICENSED",
   "scripts": {
     "start": "SKIP_PREFLIGHT_CHECK=true react-scripts start",
-    "start-app": "yarn start",
+    "start-app": "concurrently -k -s 'first' -n 'cdn,react' 'serve-cdn' 'yarn start'",
```
Comment (stephanschroeter) [inline packages/utilities-deprecated/package.json:8]: add devDependency `@porsche-design-system/assets`

File: packages/storefront/src/pages/components/typography/typography.usage.md [hunk 5]
```diff
@@ -87,17 +87,22 @@ Additional text sizes can be defined based on the Porsche type scale system.
...
+font-size: 1rem;
+line-height: {{this.fontLineHeight}}; // optimal line-height is universally defined relative to the font-size
 ```
 
-**Example Text** <p-text size="inherit" :style="typeScale(size +'px', false)">The quick brown fox jumps over the lazy
-dog</p-text>
+##### References:
+
+- [https://kittygiraudel.com/2020/05/18/using-calc-to-figure-out-optimal-line-height/](https://kittygiraudel.com/2020/05/18/using-calc-to-figure-out-optimal-line-height/)
+- [https://www.aleksandrhovhannisyan.com/blog/dont-use-a-fixed-line-height/#solution-2-use-the-ex-unit](https://www.aleksandrhovhannisyan.com/blog/dont-use-a-fixed-line-height/#solution-2-use-the-ex-unit)
```
Comment (stephanschroeter) [inline packages/storefront/src/pages/components/typography/typography.usage.md:105]: add https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#relative_length_units

**Review by denyo — APPROVED** [review]

**Pull request merged**